data/autoyast_sle15/containers/autoyast_x390s_sle15sp1.xml

```diff
@@ -0,0 +1,396 @@
+<?xml version="1.0"?>
+<!DOCTYPE profile>
+<profile xmlns="http://www.suse.com/1.0/yast2ns"
+	 xmlns:config="http://www.suse.com/1.0/configns">
+  <add-on>
+    <add_on_products config:type="list">
+      <listentry>
+        <media_url><![CDATA[https://updates.suse.com/SUSE/Products/SLE-Module-Desktop-Applications/15-SP1/s390x/product?WaKaU4v5azj4vUscFcSfSlGZwM3mBK-IPwFtBBeiQhoBCDepdbD4_s9T-TLCkRWxPVfftslVB89sh6OWyEDJqXqmJaNrbhqdJbaDODv_UhCGb4zlcmXe86pE0530Ru137Afw3xd7_Ozp0NAcUk1Qwr79hosw6__NojZKSe0H0v-L]]></media_url>
+        <product>sle-module-desktop-applications</product>
+        <product_dir/>
+      </listentry>
+      <listentry>
+        <media_url><![CDATA[https://updates.suse.com/SUSE/Products/SLE-Module-Server-Applications/15-SP1/s390x/product?ECZH2kIsyLn3gMu2bKBYatqEbgG5wwFLievb0lvVDs--P3oJPh6A-4s_o4iQuN9Jp9a1CyAsiVU36og7Mz_0KzkKAOInJbtGpnU90l5O_iiGBHvljqzPoZaYU1CKdEefjxkOmpZodr0Rrrba3E3FxGs5euufGqw5Wc71Y01jCzo]]></media_url>
```

Comment:
That links looks weird. Is this one really correct?

Reply:
it is generated by the `yast clone_system`. As well as other things. I removed a few that they do not affect the job but i did not improvize. the reason is also that this is an urgent task. Improvements can come afterwards

Reply:
Ack :+1: